app/views/scheduler/index.html.haml

```diff
@@ -9,8 +9,11 @@
           = text_field_tag :course_title, nil, placeholder: 'Add course instance', class: 'form-control'
 
         = form_for :semester, url: scheduler_path, method: :get, remote: true, html: {id: :semester_form} do |f|
-          = f.select :semester, %w(Fall Spring Summer).map { |str| [str, str.downcase] }
-          = f.select :year, options_for_select(2008..DateTime.now.year, DateTime.now.year)
+          = f.select :semester, options_for_select( |
+            %w(Fall Spring Summer).map {|str| [str, str.downcase]}, |
```

Comment:
Rubocop didn't know what to do with haml when I tried it, but it should probably work for the controller end of things - thanks!

I'm not sure what you mean with the parens - what are you proposing they wrap? The haml docs don't seem to suggest that other methods of line-breaking will work. They make it deliberately inelegant so that you're encouraged to put most of your ruby into the controller.